detox/detox.d.ts

```diff
@@ -497,6 +497,17 @@ declare global {
              * @see https://wix.github.io/Detox/docs/19.x/api/detox-object-api/#detoxtracecall
              */
             readonly traceCall: <T>(event: string, action: () => Promise<T>, args?: Record<string, unknown>) => Promise<T>;
+
+            /**
+             * Enter the REPL mode.
```

Comment:
could add link to the online REPL guide, if you write one

Reply:
Noted.